schemas/bookbrainz.sql

```diff
@@ -610,6 +610,77 @@ CREATE TABLE bookbrainz._editor_entity_visits (
 ALTER TABLE bookbrainz._editor_entity_visits ADD FOREIGN KEY (editor_id) REFERENCES bookbrainz.editor (id);
 ALTER TABLE bookbrainz._editor_entity_visits ADD FOREIGN KEY (bbid) REFERENCES bookbrainz.entity (bbid);
 
+CREATE TABLE IF NOT EXISTS bookbrainz.import (
+    id SERIAL PRIMARY KEY,
+    type bookbrainz.entity_type NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS bookbrainz.creator_import (
+	import_id INT PRIMARY KEY,
+	data_id INT
+);
+ALTER TABLE bookbrainz.creator_import ADD FOREIGN KEY (import_id) REFERENCES bookbrainz.import (id);
+ALTER TABLE bookbrainz.creator_import ADD FOREIGN KEY (data_id) REFERENCES bookbrainz.creator_data (id);
+
+CREATE TABLE IF NOT EXISTS bookbrainz.edition_import (
+	import_id INT PRIMARY KEY,
+	data_id INT
+);
+ALTER TABLE bookbrainz.edition_import ADD FOREIGN KEY (import_id) REFERENCES bookbrainz.import (id);
+ALTER TABLE bookbrainz.edition_import ADD FOREIGN KEY (data_id) REFERENCES bookbrainz.edition_data (id);
```

Comment:
Need to use `edition_import_data` with no publication/publisher fields rather than `edition_data`.

Reply:
Ok. 
But we can use the present setup too, as the fields haven't been marked `NOT NULL`. Any specific reason to not use the present table?  Upon upgradation of create entity page, creation of a new entity would not be possible without filling in the missing details. We can disable direct upgradation process for editions, and instead redirect the users to a create entity page.

Reply:
Hmm, you can use the present table for now. I think it's only nullable to allow for legacy data (where it wasn't necessary to set publication for an edition). We can just enforce this in the code, I think.